### ISSUE TITLE
collect-windows-logs.ps1 downloads script from GitHub

### ISSUE THREAD

**Issue by MikkelHegn** [Azure/aks-engine#2986]
**Describe the bug**
The [collect-windows-logs.ps1](https://github.com/Azure/aks-engine/blob/master/scripts/collect-windows-logs.ps1) script is used to gather logs and debugging information off Windows nodes. Part of this script downloads another debugging script directly from GitHub, to collect networking logs [collectlogs.ps1](https://raw.githubusercontent.com/Microsoft/SDN/master/Kubernetes/windows/debug/collectlogs.ps1)

**Expected behavior**
I would expect to not have a script download untrusted content directly off GitHub. I suggest having a signed script being present on the VHD at provision time. For customers running in locked down environments, it's not certain that there is access to the script from the Windows host.

**Additional context**
Currently this dependency is not documented for [AKS in the egress lockdown document](
https://docs.microsoft.com/en-us/azure/aks/limit-egress-traffic#required-by-windows-server-based-nodes-in-public-preview-enabled
), and we would like to to add this as an external dependency.

**Comment by MikkelHegn** [comment]
@marosset @craiglpeters - FYI

**Comment by MikkelHegn** [comment]
Self-assignment was a user error :-)

**Comment by marosset** [comment]
@MikkelHegn we currently include all of these files on the aks-windows VHDs.
Ex: https://github.com/Azure/aks-engine/blob/master/vhd/release-notes/aks-windows/2019-datacenter-core-smalldisk-17763.1098.200315.txt

There was a bug I recently fixed where files were getting moved out of the cache during deployment time instead of copied which was preventing no-egress deployments from working.
This was fixed with #2883 

We are investigating what it would take to sign these files but we are probably a few months away from that.
